pydrad/parse/parse.py

```diff
@@ -323,6 +323,19 @@ def spatial_average(self, quantity, bounds=None):
         grid_widths_bounds = self.grid_widths[i_bounds]
         return np.average(quantity_bounds, weights=grid_widths_bounds)
 
+    def column_emission_measure(self, min_logT=3.0, max_logT=8.0, dlogT = 0.05):
+        """
+            Computes the column emission measure over the whole loop
+            """
+        bins = np.arange(min_logT, max_logT+dlogT, dlogT)
```

Comment:
```suggestion
    @u.quantity_input
    def column_emission_measure(self, bins:u.K=None):
        """
        Computes the column emission measure over the whole loop
        """
        if bins is None:
            bins = 10**np.arange(3,8,0.05)*u.K
```

Reply:
I think it would be better to make the input here an array of the bin edges, in linear space rather than log space. And then that way we can enforce units on the input array as well.